src/main/java/com/lothrazar/cyclic/block/crusher/TileCrusher.java

```diff
@@ -111,6 +111,14 @@ public void tick() {
     if (level.isClientSide) {
       return;
     }
+    if (currentRecipe == null) {
+        this.findMatchingRecipe();
+        return;
+    }
+    //Checks if there is space in the output slots before processing
+    if(outputSlots.getStackInSlot(0).getCount() > 64 - currentRecipe.getResultItem(level.registryAccess()).copy().getCount()) return;
```

Comment:
fix looks good!  the hardcoded 64 can be an issue, for example if i have a recipe that outputs 3 ender pearls, that has a max stack size of 16 so it has that issue.  i can change it later you just do it something like this
 ``` 
var result = currentRecipe.getResultItem(level.registryAccess()).copy();
var max = result.getMaxStackSize();
```